tools/dind/config/services/docker.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
```

Comment:
You may want to copy unit files verbatim from https://github.com/resin-os/balena/tree/17.06-resin/contrib/init/systemd to a folder, that you never modify them, and add overrides in a separate file like you do for aufs.

Reply:
Yeah, I think it's fine to match meta-resin, and maybe I should direct this comment there as Docker's "built-in" unit files have a couple of settings that might be of interest.